Mathlib/Algebra/Category/ModuleCat/Presheaf.lean

```diff
@@ -3,15 +3,21 @@ Copyright (c) 2023 Scott Morrison All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
-import Mathlib.Algebra.Category.ModuleCat.Basic
 import Mathlib.Algebra.Category.Ring.Basic
+import Mathlib.Algebra.Category.ModuleCat.ChangeOfRings
 
 /-!
 # Presheaves of modules over a presheaf of rings.
 
-We give a hands-on description of a presheaf of modules over a fixed presheaf of rings,
+We give a hands-on description of a presheaf of modules over a fixed presheaf of rings `R`,
 as a presheaf of abelian groups with additional data.
 
+We also provide two alternative constructors :
+* `mk' : MkStruct R → PresheafOfModules R` where `M : MkStruct R` consists of a family
```

Comment:
I'm not a huge fan of `MkStruct` as a name. In other situations mathlib uses a `Core` prefix, eg to build a normed abelian group from "core data".
Relatedly, in which namespace does `MkStruct` live?

Reply:
Thanks for the review! Currently, `MkStruct` and `BundledMkStruct` are both under the `PresheafOfModules` namespace. Do you suggest something like `CorePresheafOfModules` and `BundledCorePresheafOfModules` in the root namespace? (I would slightly prefer having this under the `PresheafOfModules` namespace, like `PresheafOfModules.MkCore`, `PresheafOfModules.BundledMkCore`.) What do you think?

Reply:
I have changed this to `CorePresheafOfModules` and `BundledCorePresheafOfModules`.